app/main.py

```diff
@@ -1,8 +1,51 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self: None, comfort_class: None,
+                 clean_mark: None,
+                 brand: None) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self: None, distance_from_city_center: None,
+                 clean_power: None,
+                 average_rating: None,
+                 count_of_ratings: None) -> None:
```

Comment:
Change your type annotation to indicate what types you expect to obtain for each argument, they should not be `None` here and in other places.